### ISSUE TITLE
Training log

### ISSUE THREAD

**Issue by PhenomenalOnee** [Linzaer/Ultra-Light-Fast-Generic-Face-Detector-1MB#86]
Hello i have trained your RFB model with your data with 320x240 input
i have a less accurate model than yours
in wider_evaluate.py
your model detects a total of 13746 faces (RFB_320 model)
while mine detects 13069
can you show me your training log file 
mine is below:

[log.txt](https://github.com/Linzaer/Ultra-Light-Fast-Generic-Face-Detector-1MB/files/3884977/log.txt)

**Comment by Linzaer** [comment]
I can not find the log.txt  temporarily. I use the epoch-160 model and its validation loss is 2.8383.

**Comment by PhenomenalOnee** [comment]
Thanks for the reply
Can you tell how to add hard negatives samples in the training data

**Comment by Linzaer** [comment]
1.You can add hard negatives samples and labels to the 'Complete wideface data compression package without filtering small faces' data set I provided, then run wider_face_2_voc_add_landmark.py to generate voc format training data.
2.You need to modify the image preprocessing part to train the grayscale image.

**Comment by PhenomenalOnee** [comment]
But how to generate xml files for negative samples

**Comment by Linzaer** [comment]
If you want to reduce the false positive rate, you can try to modify the arg 'neg_pos_ratio' or  use focal loss.

**Comment by PhenomenalOnee** [comment]
OK I'ill try that
Thanks
